### PR TITLE
[AppService] Fix #11244: az webapp auth update: Add optional parameter to update client-secret-certificate-thumbprint

### LINKED ISSUE #11244: az webapp auth update command doesn't allow to set the certificate
The authSettings object has a `clientSecretCertificateThumbprint` property, which is exposed by arm templates. It is also carried in the appsettings as `WEBSITE_AUTH_CLIENT_SECRET_CERTIFICATE_THUMBPRINT`.
This setting is missing from the CLI. If you try to set this setting from the appsetting command, you get the following error `AppSetting with name 'WEBSITE_AUTH_CLIENT_SECRET_CERTIFICATE_THUMBPRINT' is not allowed.`
It'd be nice to have an additional setting name clientSecretCertificateThumbprint so we can set it up programmatically, or something similar

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: aeb196c8-e408-8563-26ba-5c3ae6e38c48
* Version Independent ID: 5f150c9a-e142-2fd7-fe84-b298dc83fd02
* Content: [az webapp auth](https://docs.microsoft.com/en-us/cli/azure/webapp/auth?view=azure-cli-latest#az-webapp-auth-update)
* Content Source: [latest/docs-ref-autogen/webapp/auth.yml](https://github.com/MicrosoftDocs/azure-docs-cli/blob/live/latest/docs-ref-autogen/webapp/auth.yml)
* Service: **app-service-web**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

### PR DESCRIPTION
**Pull request by mkarmark**
**Description**  
We want to offer support via the Azure CLI tool to update the client secret certificate thumbprint of the App Service Authentication/Authorization module. This is done by exposing a client secret certificate thumbprint parameter in the "webapp auth update" command, already offered through the Azure CLI tool.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
AppService

File: src/azure-cli/azure/cli/command_modules/appservice/custom.py [hunk 1]
```diff
@@ -847,6 +847,7 @@ def update_auth_settings(cmd, resource_group_name, name, enabled=None, action=No
                          client_id=None, token_store_enabled=None, runtime_version=None,  # pylint: disable=unused-argument
                          token_refresh_extension_hours=None,  # pylint: disable=unused-argument
                          allowed_external_redirect_urls=None, client_secret=None,  # pylint: disable=unused-argument
+                         client_secret_certificate_thumbprint=None,  # pylint: disable=unused-argument
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/appservice/custom.py:850]: May I know why it is unused?

Comment (mkarmark) [reply src/azure-cli/azure/cli/command_modules/appservice/custom.py:850]: It's sent to the app as an authSetting. I don't know why it's unused, but it's no different than the other variables which have the unused tag next to them, so I was just trying to make it match up. What does the unused argument tag mean?

Comment (qwordy) [reply src/azure-cli/azure/cli/command_modules/appservice/custom.py:850]: Pylint does not allow unused argument. The comment disables this rule.

**Pull request merged**